src/InputCell.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+export default class InputCell extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: props.row + ' - ' + props.cell,
+      row: parseInt(props.row),
+      cell : parseInt(props.cell),
+      value : 0
+    };
+    this.onInputChangeCallback = props.onChange;
+  }
+
+  render() {
+    return (
+      <input type="number" name={this.state.name} data-row={this.state.row} data-cell={this.state.cell} min="0" max="1" value={this.state.value} onChange={this.handleOnChange} aria-label="input-cell" />
```

Comment:
Are those `data` attributes just for testing?

Reply:
Yes. Could I use something more appropriate?

Reply:
Hm...

you could test the `state` values. But I am afraid the state is an implementation detail of the component. And react-testing library encourages devs to not test these details (I think it is not even possible to read the state of the component you test).

As you include `row` and `cell` in `name` it should be enough to select by the `name` value.